src/medali/lib/s1dc_flood_mapper/V1M1.ini

```diff
@@ -0,0 +1,107 @@
+# Flood mapping result metadata, version 1.0
+
+[Metadata]
+
+# creation date - date of the data processing
+date_creation: datetime
+
+# modification date - date of last modification, by default set the same as the creation time when processing the data
+date_modification: datetime
+
+# acquisition date - sensing start date
+date_sensing: datetime
+
+# dataset creator
+creator: string
+
+# name of the used worker software
+worker_name: string
+
+# tag of the used worker software
+worker_git_tag: string
+
+# 7-digit commit of the used worker software
+worker_git_commit: string
+
+# name of the used wrapper software
+wrapper_name: string
+
+# tag of the used wrapper software
+wrapper_git_tag: string
+
+# 7-digit commit of the used wrapper software
+wrapper_git_commit: string
+
+# selected run number for the processing
+run_number: integer
+
+# filename of Equi7grid tile file
+input_sar_file: string
+
+# name of used plia file (average and single-scene)
+input_plia_file: string
+
+# method used to provide a flood reference (e.g. harmonic, rolling, exponential)
+flood_reference: string
+
+# input for the used flood reference (e.g. M0 harmonic parameter file)
+input_reference: string
+
+# additional detail of the used reference method (e.g. k-value of the harmonic model)
+input_reference_detail: integer
```

Comment:
I would call it sth. like `input_reference_addon`, but please decide by yourself what you think fits better.